SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -410,7 +408,10 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 
 	if (GetDefault<USpatialGDKSettings>()->bAsyncLoadNewClassesOnEntityCheckout)
 	{
-		AsyncPackageLoadFilter = MakeUnique<SpatialGDK::AsyncPackageLoadFilter>(this);
+		AsyncPackageLoadFilter = NewObject<UAsyncPackageLoadFilter>();
+		AsyncPackageLoadFilter->Init(this);
+		AsyncPackageLoadFilter->GetOnPackageLoadedForEntityDelegate().AddDynamic(this,
```

Comment:
can we pass the function reference into init instead?

Reply:
I made the delegate a multicast so other things could listen to this event, however if we don't think that functionality is worthwhile, I can switch it to a single delegate and set via a single function.

Reply:
I would slightly prefer the single delegate since it makes the code cleaner, can also use the simple delegate instead of dynamic so we can pass `Worker_EntityId` instead of `int64`. Should be safe to do so because the delegate can only be called when the filter object is valid.